applications/plugins/SofaTest/Multi2Mapping_test.h

```diff
@@ -341,16 +341,16 @@ struct Multi2Mapping_test : public Sofa_test<typename _MultiMapping::Real>
             if (this->vectorMaxDiff(jfcIn2[p], fIn2p[p])>this->epsilon()*errorMax)
             {
                 succeed = false;
-                ADD_FAILURE() << "applyJT (parent 2) test failed" << endl << "jfcIn2[" << p << "] = " << jfcIn2[p] << endl << " fp[" << p << "] = " << fIn2p[p] << endl;
+                ADD_FAILURE() << "applyJT (parent 2) test failed" << std::endl << "jfcIn2[" << p << "] = " << jfcIn2[p] << std::endl << " fp[" << p << "] = " << fIn2p[p] << std::endl;
             }
         }
 
         // ================ test getJs() : check that J.vp = vc
         if (this->vectorMaxDiff(Jv, vc) > this->epsilon()*errorMax){
             succeed = false;
-            msg_info() << "Jvp = " << Jv;
-            msg_info() << "vc  = " << vc;
-            ADD_FAILURE() << "getJs() test failed" << endl << "Jvp = " << Jv << endl << "vc  = " << vc << endl;
+            std::cout << "Jvp = " << Jv << std::endl;
```

Comment:
Any reason why swapping the msg_info for a cout?

Reply:
There is very strong argument to use msg_info/msg_error.